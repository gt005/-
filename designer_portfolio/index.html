<!doctype html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport"
          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Denis Novik</title>
    <link rel="stylesheet" href="css/main.css">
    <link rel="stylesheet" href="css/bootstrap.min.css">
</head>
<body>

<header class="header">
    <div class="container">
        <div class="row">
            <div class="col-md-12">
                <input type="checkbox" id="menu-mobile-toggle-btn">
                <label for="menu-mobile-toggle-btn" class="show-menu-mobile-btn">
                    <img src="img/menu-btn.svg" alt="menu" class="open-menu-btn">
                </label>
                <nav class="menu-nav">
                    <ul class="menu">
                        <li class="menu-item">
                            <a class="active" href="#home">
                                Home
                            </a>
                        </li>
                        <li class="menu-item">
                            <a href="#about-me">
                                About me
                            </a>
                        </li>
                        <li class="menu-item">
                            <a href="#skills">
                                Skills
                            </a>
                        </li>
                        <li class="menu-item">
                            <a href="#portfolio">
                                Portfolio
                            </a>
                        </li>
                        <li class="menu-item">
                            <a href="#contacts">
                                Contacts
                            </a>
                        </li>
                        <label for="menu-mobile-toggle-btn" class="hide-menu-mobile-btn">
                            <img src="img/close-menu-btn.svg" alt="close" class="close-menu-btn">
                        </label>
                        <div class="mobile-language-toggle-container">
                            <p class="mobile-language-toggle">
                                <span class="ru-toggle">RU</span> | <span class="eng-toggle">ENG</span>
                            </p>
                        </div>
                    </ul>
                </nav>
            </div>
        </div>
        <div class="row header-title" id="home">
            <div class="col-md-2">
                <h1 class="header-title__author-name">
                    Denis
                    Novik
                </h1>
            </div>
            <div class="col-md-3">
                <p class="header-title__author-description">
                    UX | UI designer <br>
                    24 years old, Minsk
                </p>
            </div>
            <div class="col-md-2 pr-md-0">
                <div class="header-title__language-toggle-container">
                    <p class="header-title__language-toggle">
                        <span class="ru-toggle">RU</span> | <span class="eng-toggle">ENG</span>
                    </p>
                </div>
            </div>
        </div>
        <div class="row">
            <div class="col-12">
                <div class="author-photo">

                </div>
            </div>
        </div>
    </div>
</header>

<section class="about-me" id="about-me">
    <div class="container">
        <div class="row">
            <div class="col-12 text-center justify-content-center">
                <div class="about-me-text">
                    <h2 class="about-me-text__title">
                        About me
                    </h2>
                    <p class="about-me-text__text">
                        Hi, I'm Denis â€“ UX/UI designer from Minsk. <br>
                        I'm interested in design and everything connected with it.
                    </p>
                    <p class="about-me-text__text">
                        I'm studying at courses "Web and mobile design <br>
                        interfaces" in IT-Academy.
                    </p>
                    <p class="about-me-text__text">
                        Ready to implement excellent projects <br>
                        with wonderful people.
                    </p>
                </div>
            </div>
        </div>
    </div>
</section>

<section class="skills">
    <div class="container">
        <div class="row">
            <div class="col-12 text-center justify-content-center">
                <h2 class="skills-title__title">
                    Skills
                </h2>
                <p class="skills-title__text">
                    I work in such programs as
                </p>
            </div>
        </div>
        <div class="row justify-content-between">
            <div class="col-md-3">
                <div class="skill-card mt-0">
                    <img src="img/adobe-Photoshop-icon.svg" alt="adobe Photoshop" class="skill-card__img">
                    <p class="skill-card__name">
                        Adobe <br>
                        Photoshop
                    </p>
                    <div class="skill-card__rating">
                        <img src="img/Star.svg" alt="star">
                        <img src="img/Star.svg" alt="star">
                        <img src="img/Star.svg" alt="star">
                        <img src="img/Star.svg" alt="star">
                        <img src="img/Star.svg" alt="star" class="skill-card__rating-not-active-star">
                    </div>
                </div>
            </div>
            <div class="col-md-3">
                <div class="skill-card">
                    <img src="img/adobe-illustrator-icon.svg" alt="adobe illustrator" class="skill-card__img">
                    <p class="skill-card__name">
                        Adobe <br>
                        Illustrator
                    </p>
                    <div class="skill-card__rating">
                        <img src="img/Star.svg" alt="star">
                        <img src="img/Star.svg" alt="star">
                        <img src="img/Star.svg" alt="star">
                        <img src="img/Star.svg" alt="star" class="skill-card__rating-not-active-star">
                        <img src="img/Star.svg" alt="star" class="skill-card__rating-not-active-star">
                    </div>
                </div>
            </div>
            <div class="col-md-3">
                <div class="skill-card">
                    <img src="img/adobe-after-effects-icon.svg" alt="adobe after effects" class="skill-card__img">
                    <p class="skill-card__name">
                        Adobe <br>
                        After Effects
                    </p>
                    <div class="skill-card__rating">
                        <img src="img/Star.svg" alt="star">
                        <img src="img/Star.svg" alt="star">
                        <img src="img/Star.svg" alt="star">
                        <img src="img/Star.svg" alt="star">
                        <img src="img/Star.svg" alt="star" class="skill-card__rating-not-active-star">
                    </div>
                </div>
            </div>
            <div class="col-md-3">
                <div class="skill-card">
                    <img src="img/figma-icon.svg" alt="figma" class="skill-card__img">
                    <p class="skill-card__name">
                        Figma
                    </p>
                    <div class="skill-card__rating">
                        <img src="img/Star.svg" alt="star">
                        <img src="img/Star.svg" alt="star">
                        <img src="img/Star.svg" alt="star">
                        <img src="img/Star.svg" alt="star">
                        <img src="img/Star.svg" alt="star" class="skill-card__rating-not-active-star">
                    </div>
                </div>
            </div>
        </div>
    </div>
</section>

<section class="portfolio">
    <div class="container">
        <div class="row">
            <div class="col-12">
                <h2 class="portfolio-title">
                    Portfolio
                </h2>
            </div>
        </div>
        <div class="row">
            <div class="col-12">
                <div class="portfolio-work">
                    <img src="img/portfolio-work-1.jpg" class="portfolio-work__photo" alt="portfolio photo">
                    <p class="portfolio-work__name">
                        Online fashion store - Homepage
                    </p>
                </div>
            </div>
        </div>
        <div class="row">
            <div class="col-12">
                <div class="portfolio-work">
                    <img src="img/portfolio-work-2.jpg" class="portfolio-work__photo" alt="portfolio photo">
                    <p class="portfolio-work__name">
                        Reebok Store - Concept
                    </p>
                </div>
            </div>
        </div>
        <div class="row">
            <div class="col-12">
                <div class="portfolio-work">
                    <img src="img/portfolio-work-3.jpg" class="portfolio-work__photo" alt="portfolio photo">
                    <p class="portfolio-work__name">
                        Braun Landing Page - Concept
                    </p>
                </div>
            </div>
        </div>
    </div>
</section>

<footer class="footer">
    <div class="container">
        <div class="row">
            <div class="col-12 d-flex flex-column align-items-center">
                <h2 class="contacts-title">
                    Contacts
                </h2>
                <p class="contacts-title-text">
                    Want to know more or just chat? <br>
                    You are welcome!
                </p>
                <a href="#" class="contacts-send-btn">
                    Send message
                </a>
                <div class="contacts-social-networks">
                    <img src="img/linkedin-icon.svg" alt="linkedin">
                    <img src="img/instagram-icon.svg" alt="instagram">
                    <img src="img/behance-icon.svg" alt="behance">
                    <img src="img/dribble-icon.svg" alt="dripple">
                </div>
                <p class="contacts-social-networks-description">
                    Like me on <br>
                    LinkedIn, Instagram, Behance, Dribble
                </p>
            </div>
        </div>
    </div>
</footer>

<script src="js/jquery-3.5.1.min.js"></script>
<script src="js/main.js"></script>

</body>
</html>