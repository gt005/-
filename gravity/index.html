<!doctype html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Gravity</title>
    <meta name="viewport"
          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link href="https://fonts.googleapis.com/css2?family=Fira+Sans:wght@400;500;700&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="css/bootstrap.min.css">
    <link rel="stylesheet" href="css/owl.carousel.min.css">
    <link rel="stylesheet" href="css/owl.theme.default.min.css">
    <link rel="stylesheet" href="css/main.css">
</head>
<body>

<header class="header">
    <div class="nav">
        <div class="container">
            <div class="row">
                <div class="col-2">
                    <img src="img/logo.svg" alt="logo" class="logo">
                </div>
                <div class="offset-4 col-4">
                    <button>
                        <img src="img/search.svg" alt="search">
                    </button>
                </div>
                <div class="col-2">
                    <button class="ml-auto">
                        <img src="img/menu.svg" alt="menu">
                    </button>
                </div>
            </div>
        </div>
    </div>
    <div class="offer">
        <div class="container">
            <div class="row">
                <div class="col-md-6">
                    <div class="offer-cont">
                        <h1 class="offer-cont__title">
                            Work around you and your team
                        </h1>
                        <p class="offer-cont__text">
                            From ads that dance or sing to MTV-like commercials, online advertisers are now using a new type of technology “rich media” to attract consumers.
                        </p>
                        <button class="btn">
                            Get started
                        </button>
                    </div>
                </div>
                <div class="col-md-6 be-static">
                    <div class="hero">
                        <a href="#" class="video-btn"></a>
                    </div>
                </div>
            </div>
        </div>
    </div>
</header>

<section class="about">
    <div class="container">
        <div class="row">
            <div class="col-md-5">
                <img src="img/img2.jpg" alt="about poster">
            </div>
            <div class="offset-1 col-md-6 d-flex align-items-center">
                <div class="about-cont">
                    <h2 class="about-cont__title">
                        We solve digital problems with an outstanding creative flare
                    </h2>
                    <p class="about-cont__text">
                        The best time to view the moon, obviously, is at night when there are few clouds and the weather is accommodating for a long and lasting study.
                    </p>
                    <p class="about-cont__text">
                        For most of us, the idea of astronomy is something we directly connect to “stargazing”, telescopes and seeing magnificent displays in the heavens.
                    </p>
                    <div class="author">
                        <img src="img/author.jpg" alt="author image">
                        <div class="about-author">
                            <h1 class="author-name">
                                Denis Shepovalov
                            </h1>
                            <p class="author-work">
                                Co-Founder & CEO
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </div>
</section>

<section class="services">
    <div class="container">
        <div class="row">
            <div class="col-md-6">
                <div class="services-title">
                    <p class="services-title__label">
                        — Services
                    </p>
                    <h2 class="services-title__header">
                        We love work hard and explore new things that’s why we cover lot of things. Check our service list and view details
                    </h2>
                </div>
            </div>
            <div class="col-md-5 ml-auto d-flex justify-content-between">
                <ul class="services-list services-list__left">
                    <li class="services-list__item">Multimedia</li>
                    <li class="services-list__item">Interactive design</li>
                    <li class="services-list__item">Packaging</li>
                    <li class="services-list__item">Branding</li>
                </ul>
                <ul class="services-list services-list__right">
                    <li class="services-list__item">Creative Strategy</li>
                    <li class="services-list__item">UI/UX</li>
                    <li class="services-list__item">Coding</li>
                </ul>
            </div>
        </div>
        <div class="row">
            <div class="col">
                <div class="owl-carousel owl-theme">
                    <div class="item">
                        <img class="services-play" src="img/play.svg" alt="play">
                        <img src="img/img3.jpg" alt="work">
                    </div>
                    <div class="item">
                        <img class="services-play" src="img/play.svg" alt="play">
                        <img src="img/img4.jpg" alt="work">
                    </div>
                </div>
            </div>
        </div>
    </div>

</section>

<section class="features">
    <div class="container">
        <div class="row">
            <div class="col-md-6 be-static">
                <div class="feature-images">
                    <img src="img/img5.jpg" alt="features poster" class="features-image">
                </div>
            </div>
            <div class="col-md-6">
                <div class="features-cont">
                    <h2 class="features-cont__title">
                        Well thought with super easy to use
                    </h2>
                    <p class="features-cont__text">
                        As we approached the palace I could see through the great windows of the first floor the brilliantly illuminated audience chamber of Than Kosis. The immense hall was crowded with nobles and their women.
                    </p>
                    <ul class="features-cont__main__features">
                        <li class="features-cont__main__feature">
                            <img class="features-cont__main__feature__image" src="img/icon100.svg" alt="Ultimate Result">
                            Ultimate Result
                        </li>
                        <li class="features-cont__main__feature">
                            <img class="features-cont__main__feature__image" src="img/bag.svg" alt="minimal design">
                            Minimal Design
                        </li>
                    </ul>
                </div>
            </div>
        </div>
    </div>
</section>

<section class="works">
    <div class="container">
        <div class="row">
            <div class="col-md-6">
                <div>
                    <p class="section-works-title">
                        — our works
                    </p>
                    <h2 class="works-title">
                        Discover our Work
                    </h2>
                </div>
            </div>
            <div class="col-md-6">
                <ul class="nav nav-pills mb-3" id="pills-tab" role="tablist">
                    <li class="nav-item" role="presentation">
                        <a class="work-nav nav-link active" id="pills-all-tab" data-toggle="pill" href="#pills-all" role="tab" aria-controls="pills-contact" aria-selected="true">all</a>
                    </li>
                    <li class="nav-item" role="presentation">
                        <a class="work-nav nav-link" id="pills-branding-tab" data-toggle="pill" href="#pills-branding" role="tab" aria-controls="pills-branding" aria-selected="false">branding</a>
                    </li>
                    <li class="nav-item" role="presentation">
                        <a class="work-nav nav-link" id="pills-web-tab" data-toggle="pill" href="#pills-web" role="tab" aria-controls="pills-web" aria-selected="false">web</a>
                    </li>
                    <li class="nav-item" role="presentation">
                        <a class="work-nav nav-link" id="pills-development-tab" data-toggle="pill" href="#pills-development" role="tab" aria-controls="pills-development" aria-selected="false">development</a>
                    </li>
                    <li class="nav-item" role="presentation">
                        <a class="work-nav nav-link" id="pills-films-tab" data-toggle="pill" href="#pills-films" role="tab" aria-controls="pills-films" aria-selected="false">films</a>
                    </li>

                </ul>
            </div>
        </div>
        <div class="row tab-content" id="pills-tabContent">
            <div class="col">
                <div class="works-wrap tab-pane fade show active" id="pills-all" role="tabpanel" aria-labelledby="pills-all-tab">
                    <a href="" class="works-wrap__item"><img src="img/img7.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img8.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img9.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img10.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img11.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img12.jpg" alt="work"></a>
                </div>
                <div class="works-wrap tab-pane fade" id="pills-branding" role="tabpanel" aria-labelledby="pills-branding-tab">
                    <a href="" class="works-wrap__item"><img src="img/img8.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img7.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img10.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img9.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img12.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img11.jpg" alt="work"></a>
                </div>
                <div class="works-wrap tab-pane fade" id="pills-web" role="tabpanel" aria-labelledby="pills-web-tab">
                    <a href="" class="works-wrap__item"><img src="img/img11.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img8.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img7.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img9.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img10.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img12.jpg" alt="work"></a>
                </div>
                <div class="works-wrap tab-pane fade" id="pills-development" role="tabpanel" aria-labelledby="pills-development-tab">
                    <a href="" class="works-wrap__item"><img src="img/img10.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img8.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img12.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img11.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img7.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img9.jpg" alt="work"></a>
                </div>
                <div class="works-wrap tab-pane fade" id="pills-films" role="tabpanel" aria-labelledby="pills-films-tab">
                    <a href="" class="works-wrap__item"><img src="img/img12.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img11.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img10.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img9.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img8.jpg" alt="work"></a>
                    <a href="" class="works-wrap__item"><img src="img/img7.jpg" alt="work"></a>
                </div>
            </div>
        </div>
        <div class="row works-end-text">
            <div class="col-md-8">
                <p class="work-text">
                    It picks up the words on the page and displays ads that are similar to those words. Then when someone either performs an action or clicks on your page you will get paid.
                </p>
            </div>
            <div class="col-md-4 work-end-btn">
                <button class="btn">
                    see our works
                </button>
            </div>
        </div>
    </div>
</section>

<script src="js/jquery-3.5.1.min.js"></script>
<script src="js/owl.carousel.min.js"></script>
<script src="js/bootstrap.min.js"></script>
<script src="js/main.js"></script>

</body>
</html>