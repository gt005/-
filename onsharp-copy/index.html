<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport"
          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <meta name="msapplication-TileImage" content="img/Onsharp_favicon.ico">
    <title>Web Design, Apps, and Custom Development in Fargo ND | Onsharp</title>
    <link rel="stylesheet" href="css/bootstrap.min.css">
    <link rel="stylesheet" id="siteorigin-google-web-fonts-css" href="//fonts.googleapis.com/css?family=Montserrat%3A400%2C500%2C600%2C700%7COpen+Sans%3A300%2C400%2C600&amp;ver=5.4.2" type="text/css" media="all">
    <link rel="icon" href="img/Onsharp_favicon.ico" sizes="32x32">
    <link rel="icon" href="img/Onsharp_favicon.ico" sizes="192x192">
    <link rel="apple-touch-icon" href="img/Onsharp_favicon.ico">
    <link rel="stylesheet" href="css/style.css">
    <script src="https://kit.fontawesome.com/65f48fc481.js" crossorigin="anonymous"></script>
</head>


<header class="site-header">
    <div class="container-md">
        <div class="row">
            <nav class="navbar navbar-expand-md p-0 w-100">
                <div class="col-md-3 pl-0">
                    <a class="navbar-brand p-0" href="#">
                        <img src="img/onsharp-logo-min.png" alt="Onsharp" width="300" height="73" class="header-logo">
                    </a>
                </div>

                <div class="col-md-9 pr-0">
                    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
                        <span class="navbar-toggler-icon"></span>
                    </button>
                    <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
                        <div class="navbar-nav ml-auto align-items-center">
                            <a class="nav-item nav-link main-navigation-link" href="#">WHAT WE DO</a>
                            <a class="nav-item nav-link main-navigation-link" href="#">WHO WE SERVE</a>
                            <a class="nav-item nav-link main-navigation-link" href="#">OUR WORK</a>
                            <a class="nav-item nav-link main-navigation-link" href="#">ABOUT</a>
                            <a class="nav-item nav-link main-navigation-link" href="#">RESOURCES</a>
                            <a class="nav-item nav-link main-navigation-link main-button" href="#">WORK WITH US</a>
                        </div>
                    </div>
                </div>

            </nav>
        </div>

    </div>
</header>

<section class="header-section">
    <div class="container">
        <div class="row">
            <div class="col pl-0 pr-0">
                <div class="header-title__label">
                    <span style="color: #ffffff">DESIGN.</span>
                    <span style="color: #8b9b93">BUILD.</span>
                    <span style="color: #f58025">GROW.</span>
                </div>
                <p class="header-title__text">
                    We design and build high-quality mobile apps, websites, and custom software solutions.
                </p>
                <a href="#" class="main-button">REQUEST A CONSULTATION</a>
            </div>
        </div>
    </div>
</section>


<section class="what-we-do">
    <div class="container p-0">
        <div class="row">
            <div class="col pr-0 pl-0">
                <div class="what-we-do-title">
                    <h3 class="what-we-do-title__label">
                        WHAT WE DO
                    </h3>
                    <p class="what-we-do-title__text">
                        We are experts at designing and building websites, mobile apps, and custom software solutions. We know that every business faces different and unique challenges, and we're here to guide you in the right direction.
                    </p>
                </div>
            </div>
        </div>
        <div class="row">
            <ul class="what-we-do-cards">
                <li class="what-we-do-card-item col-md-4">
                    <i class="fab fa-dev"></i>
                    <a href="#" class="card-link">
                        App & Software <br>
                        Development
                    </a>
                    <p class="what-card-text">
                        We use the latest technologies to build apps and custom software solutions.
                    </p>
                    <a href="#" class="what-card-learn-more">
                        Learn More
                    </a>
                </li>

                <li class="what-we-do-card-item col-md-4">
                    <i class="fab fa-chrome"></i>
                    <a href="#" class="card-link">
                        Web Design <br>
                        & Hosting
                    </a>
                    <p class="what-card-text">
                        We design, build, and host websites for hundreds of innovative companies.
                    </p>
                    <a href="#" class="what-card-learn-more">
                        Learn More
                    </a>
                </li>

                <li class="what-we-do-card-item col-md-4">
                    <i class="fas fa-camera"></i>
                    <a href="#" class="card-link">
                        Web Photos <br>
                        & Videos
                    </a>
                    <p class="what-card-text">
                        We capture brilliant photos and videos that boost the visual appearance of your website.
                    </p>
                    <a href="#" class="what-card-learn-more">
                        Learn More
                    </a>
                </li>
            </ul>
        </div>
    </div>
</section>

<section class="location-numbers">
    <div class="container">
        <div class="row d-flex justify-content-between text-center">
            <div class="col-md-4 p-0">
                <div class="location-number">
                    3
                </div>
                <div class="location-text">
                    OFFICE LOCATIONS
                </div>
            </div>
            <div class="col-md-4 p-0">
                <div class="location-number">
                    18
                </div>
                <div class="location-text">
                    STATES WHERE WE SERVE CLIENTS
                </div>
            </div>
            <div class="col-md-4 p-0">
                <div class="location-number">
                    250+
                </div>
                <div class="location-text">
                    ACTIVE CLIENTS
                </div>
            </div>
        </div>
    </div>
</section>

<section class="serve">
    <div class="container p-0">
        <div class="row">
            <div class="col-md-6">
                <div class="serve-title-block">
                    <h2 class="serve-title__label">
                        WHO WE <span style="color:#f58025;">SERVE</span>
                    </h2>
                    <p class="serve-title__text">
                        With offices in Fargo, Minneapolis, and Denver, we work with 250+ clients throughout the Midwest and across the country.
                    </p>
                    <hr class="serve-title__line">
                    <a href="#" class="main-button serve-title__btn">
                        Download Our Brochure
                    </a>
                </div>
            </div>
            <div class="col-md-6">
                <img src="img/Client_Map_Transparent_BG-min.png" alt="country" class="serve-title__image">
            </div>
        </div>


        <div style="margin-top: 110px" class="serv-card-row row d-flex justify-content-between">
            <div class="col-md-4">
                <div class="serve-card">
                    <div class="serve-image">
                        <i class="fas fa-circle serve-image-bg"></i>
                        <img src="img/crops_white-min.png" alt="dollars-symbol_white" class="serve-image-front">
                    </div>
                    <a href="#" class="card-link">
                        Agriculture
                    </a>
                    <p class="serve-card-text">
                        Make sure clients and farmers understand who you are and what you do with an intuitive website that easily connects with your audience.
                    </p>
                    <a href="#" class="what-card-learn-more">
                        Learn More
                    </a>
                </div>
            </div>
            <div class="col-md-4">
                <div class="serve-card">
                    <div class="serve-image">
                        <i class="fas fa-circle serve-image-bg"></i>
                        <img src="img/dollars-symbol_white-min_4.png" alt="dollars-symbol_white" class="serve-image-front">
                    </div>
                    <a href="#" class="card-link">
                        Banking & Finance
                    </a>
                    <p class="serve-card-text">
                        With decades of experience and a deep understanding of security and compliance, Onsharp has your best interest in mind.
                    </p>
                    <a href="#" class="what-card-learn-more">
                        Learn More
                    </a>
                </div>
            </div>
            <div class="col-md-4">
                <div class="serve-card">
                    <div class="serve-image">
                        <i class="fas fa-circle serve-image-bg"></i>
                        <img src="img/traffic-cone_white-min.png" alt="traffic-cone_white" class="serve-image-front">
                    </div>

                    <a href="#" class="card-link">
                        Construction
                    </a>
                    <p class="serve-card-text">
                        Creating a website is a lot like construction. Let us help you build an online presence that truly represents the quality of your work.
                    </p>
                    <a href="#" class="what-card-learn-more">
                        Learn More
                    </a>
                </div>
            </div>
        </div>

        <div style="margin-top: 110px" class="serv-card-row row d-flex justify-content-between">
            <div class="col-md-4">
                <div class="serve-card">
                    <div class="serve-image">
                        <i class="fas fa-circle serve-image-bg"></i>
                        <img src="img/compass-for-drawing_white-min.png" alt="dollars-symbol_white" class="serve-image-front">
                    </div>
                    <a href="#" class="card-link">
                        Engineering
                    </a>
                    <p class="serve-card-text">
                        Don't let your website undermine what you do as an engineering firm. Let us help you demonstrate your hard work and dedication to each project.
                    </p>
                    <a href="#" class="what-card-learn-more">
                        Learn More
                    </a>
                </div>
            </div>
            <div class="col-md-4">
                <div class="serve-card">
                    <div class="serve-image">
                        <i class="fas fa-circle serve-image-bg"></i>
                        <img src="img/Healthcare.png" alt="dollars-symbol_white" class="serve-image-front">
                    </div>
                    <a href="#" class="card-link">
                        Healthcare
                    </a>
                    <p class="serve-card-text">
                        We have more than 19 years of experience building healthcare websites, management systems, and billing integrations.
                    </p>
                    <a href="#" class="what-card-learn-more">
                        Learn More
                    </a>
                </div>
            </div>
            <div class="col-md-4">
                <div class="serve-card">
                    <div class="serve-image">
                        <i class="fas fa-circle serve-image-bg"></i>
                        <img src="img/tools-cross-settings-symbol-for-interface_white-min_4.png" alt="traffic-cone_white" class="serve-image-front">
                    </div>

                    <a href="#" class="card-link">
                        Manufacturing
                    </a>
                    <p class="serve-card-text">
                        Display your great work, process, and attention to detail on your website so people can understand what makes you unique.
                    </p>
                    <a href="#" class="what-card-learn-more">
                        Learn More
                    </a>
                </div>
            </div>
        </div>

        <div style="margin-top: 110px" class="serv-card-row row d-flex justify-content-between">
            <div class="col-md-4">
                <div class="serve-card">
                    <div class="serve-image">
                        <i class="fas fa-circle serve-image-bg"></i>
                        <img src="img/RealEstate.png" alt="dollars-symbol_white" class="serve-image-front">
                    </div>
                    <a href="#" class="card-link">
                        Real Estate
                    </a>
                    <p class="serve-card-text">
                        We provide innovate web design and software integration services that help our real estate clients better serve their tenants.
                    </p>
                    <a href="#" class="what-card-learn-more">
                        Learn More
                    </a>
                </div>
            </div>
            <div class="col-md-4">
                <div class="serve-card">
                    <div class="serve-image">
                        <i class="fas fa-circle serve-image-bg"></i>
                        <img src="img/Transport-Logistics-min.png" alt="dollars-symbol_white" class="serve-image-front">
                    </div>
                    <a href="#" class="card-link">
                        Transportation & Logistics
                    </a>
                    <p class="serve-card-text">
                        We help our clients leverage new and emerging transportation and logistics technologies that drive growth and efficiency.
                    </p>
                    <a href="#" class="what-card-learn-more">
                        Learn More
                    </a>
                </div>
            </div>
            <div class="col-md-4">
                <div class="serve-card">
                    <div class="serve-image">
                        <i class="fas fa-circle serve-image-bg"></i>
                        <img src="img/electric-tower_white-min.png" alt="traffic-cone_white" class="serve-image-front">
                    </div>

                    <a href="#" class="card-link">
                        Utilities
                    </a>
                    <p class="serve-card-text">
                        Stay connected to your customers with a website that represents who you are and what you do. Let us help you build a solution that works for you.
                    </p>
                    <a href="#" class="what-card-learn-more">
                        Learn More
                    </a>
                </div>
            </div>
        </div>
    </div>
</section>

<section class="subscribe">
    <div class="container">
        <div class="row">
            <div class="col d-flex flex-column justify-content-center">
                <h2 class="subscribe-title">
                    Valuable Insights
                </h2>
                <p class="subscribe-text">
                    Subscribe to our newsletter for valuable web design and app development tips that will help you run a more successful business.
                </p>
            </div>
        </div>
        <div class="row" style="margin-top: 40px;">
            <div class="offset-4 col-md-4">
                <a href="#" class="subscribe-btn">
                    Sign Me Up!
                </a>
            </div>
        </div>
    </div>
</section>


<section class="companies">
    <div class="container p-0">
        <div class="row">
            <div class="col pr-0 pl-0">
                <div class="what-we-do-title">
                    <h3 class="what-we-do-title__label">
                        OUR WORK GETS NOTICED
                    </h3>
                    <p class="what-we-do-title__text">
                        We work with innovative companies in agriculture, banking & finance, construction, engineering, manufacturing, and utilities.
                    </p>
                </div>
            </div>
        </div>
        <div class="row">
            <div class="offset-md-1 col-md-2">
                <a href="#" class="company-card">
                    <img src="#" alt="company" class="company-image">
                </a>
            </div>
            <div class="col-md-2">
                <a href="#" class="company-card">
                    <img src="#" alt="company" class="company-image">
                </a>
            </div>
            <div class="col-md-2">
                <a href="#" class="company-card">
                    <img src="#" alt="company" class="company-image">
                </a>
            </div>
            <div class="col-md-2">
                <a href="#" class="company-card">
                    <img src="#" alt="company" class="company-image">
                </a>
            </div>
            <div class="col-md-2 mr-auto">
                <a href="#" class="company-card">
                    <img src="#" alt="company" class="company-image">
                </a>
            </div>
        </div>
    </div>

</section>

<script src="js/bootstrap.min.js"></script>

</body>
</html>