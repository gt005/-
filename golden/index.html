<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <link rel="stylesheet" href="css/style.css">
    <title>Golden</title>
</head>
<body>

<header>
    <div class="container">
        <nav class="nav-bar-container">
            <img src="img/logo.png" alt="logo" class="logo">
            <ul class="nav-bar">
                <li><a class="nav-link" href="#">home</a></li>
                <li><a class="nav-link" href="#">services</a></li>
                <li><a class="nav-link" href="#">portfolio</a></li>
                <li><a class="nav-link" href="#">about</a></li>
                <li><a class="nav-link" href="#">contact</a></li>
            </ul>
        </nav>

        <h1>welcome to our studio!</h1>
        <h1>it's nice to meet you</h1>
        <button>tell me more</button>
    </div>
</header>
<section>

</section>

<section></section>
<section></section>
</body>
</html>